zendframework/zendframework/ZF2015-06.yaml

```diff
@@ -0,0 +1,23 @@
+title:     XXE/XEE vector when using ZendXml on multibyte payloads
+link:      http://framework.zend.com/security/advisory/ZF2015-06
+cve:       CVE-2015-5161
+branches:
+    2.0.x:
+        time:     2015-08-03 15:13:58
+        versions: [>=2.0.0,<2.0.99]
+    2.1.x:
+        time:     2015-08-03 15:13:58
+        versions: [>=2.1.0,<2.1.99]
+    2.2.x:
+        time:     2015-08-03 15:13:58
+        versions: [>=2.2.0,<2.2.99]
+    2.3.x:
+        time:     2015-08-03 15:13:58
+        versions: [>=2.3.0,<2.3.8]
+    2.4.x:
+        time:     2015-08-03 15:13:58
+        versions: [>=2.4.0,<2.4.6]
+    2.5.x:
+        time:     2015-08-03 15:13:58
+        versions: [>=2.5.0,<2.5.1]
```

Comment:
@stof Then I respectfully ask that you change the README to reflect that the repository is only for:

- Machine readable vulnerability information, that
- is only usable with packages installable via composer
- and specifically referenced in packagist.org.

Right now, the repository README file indicates the following:

> The PHP Security Advisories Database references known security vulnerabilities in various PHP projects and libraries. This database must not serve as the primary source of information for security issues, it is not authoritative for any referenced software, but it allows to centralize information for convenience and easy consumption.

It specifically indicates that it is a *database*, and indicates that you can determine if a version is vulnerable in *multiple* ways:

- Via an online, browseable database
- A CLI tool
- An API

Nowhere does it indicate that the vulnerabilities listed are limited in the fashion I suggested above. As such, I have listed the vulnerabilities *as we have reported them and mitigated them in our project.* 

I'm leaving the PR as-is unless changes to the README to limit the scope of this repository are made.